langserve/server.py

```diff
@@ -456,204 +457,163 @@ def _json_encode_response(model: BaseModel) -> JSONResponse:
-    well_known_lc_serializer = WellKnownLCSerializer()
+    def __init__(
+        self,
+        runnable: Runnable,
+        *,
+        path: str = "",
+        base_url: str = "",
+        input_type: Union[Type, Literal["auto"], BaseModel] = "auto",
+        output_type: Union[Type, Literal["auto"], BaseModel] = "auto",
+        config_keys: Sequence[str] = ("configurable",),
+        include_callback_events: bool = False,
+        enable_feedback_endpoint: bool = False,
+        per_req_config_modifier: Optional[PerRequestConfigModifier] = None,
+    ) -> None:
+        """Create a new RunnableServer.
+
+        Args:
+            runnable: The runnable to serve.
+            path: The path to serve the runnable under.
+            base_url: Base URL for playground
+            input_type: type to use for input validation.
+                Default is "auto" which will use the InputType of the runnable.
+                User is free to provide a custom type annotation.
+                Favor using runnable.with_types(input_type=..., output_type=...)
+                instead. This parameter may get deprecated!
+            output_type: type to use for output validation.
+                Default is "auto" which will use the OutputType of the runnable.
+                User is free to provide a custom type annotation.
+                Favor using runnable.with_types(input_type=..., output_type=...)
+                instead. This parameter may get deprecated!
+            config_keys: list of config keys that will be accepted, by default
+                will accept `configurable` key in the config. Will only be used
+                if the runnable is configurable. Cannot configure run_name,
+                which is set by default to the path of the API.
+            include_callback_events: Whether to include callback events in the response.
+                If true, the client will be able to show trace information
+                including events that occurred on the server side.
+                Be sure not to include any sensitive information in the callback events.
+            enable_feedback_endpoint: Whether to enable an endpoint for logging feedback
+                to LangSmith. Enabled by default. If this flag is disabled or LangSmith
+                tracing is not enabled for the runnable, then 400 errors will be thrown
+                when accessing the feedback endpoint
+            per_req_config_modifier: optional function that can be used to update the
+                RunnableConfig for a given run based on the raw request. This is useful,
+                for example, if the user wants to pass in a header containing
+                credentials to a runnable. The RunnableConfig is presented in its
+                dictionary form. Note that only keys in `config_keys` will be
+                modifiable by this function.
+        """
+        if importlib.util.find_spec("sse_starlette") is None:
```

Comment:
dumb python question, but why can't we enforce that our users have to install sse_starlette when using langserve?